ibis/expr/datatypes/core.py

```diff
@@ -177,6 +224,9 @@ def is_null(self) -> bool:
     def is_numeric(self) -> bool:
         return isinstance(self, Numeric)
 
+    def is_parametric(self) -> bool:
```

Comment:
Is this method used anywhere, or do you have a plan to use it somewhere?

Reply:
Indeed, we shouldn't expose it. Removed.

Reply:
Also made the Parametric class private to the module.